frontend/src/sections/EventSection/EventCard.tsx

```diff
@@ -2,7 +2,7 @@
 
 import React from 'react'
 import Image from 'next/image'
-import dayjs from 'dayjs'
+import dayjsWithTZ from '@/utils/dayjsWithTZ'
```

Comment:
```suggestion
import dayjsWithTZ from '@utils'
```

This will work after we export from barrel file.